pkg/apis/core/types.go

```diff
@@ -2971,8 +2971,6 @@ type PodCondition struct {
 type PodResizeStatus string
 
 const (
-	// Pod resources resize has been requested and will be evaluated by node.
-	PodResizeStatusProposed PodResizeStatus = "Proposed"
```

Comment:
I'm not sure if there's any reason to keep this as a deprecated value... if not, you should also remove it from the v1 types.

I'll defer to the api reviewer on whether to keep it or not.

Reply:
/assign @liggitt 
for api review

Reply:
just drop it